### PR TITLE
feat: added skin category

### PR DESCRIPTION
**Pull request by cazala**
Added `skin` category to subgraph + webapp

### CODE REVIEW

File: webapp/src/modules/translation/locales/es.json [hunk 1]
```diff
@@ -88,6 +88,7 @@
     "wearables_mask": "Mascaras",
     "wearables_tiara": "Tiara",
     "wearables_top_head": "Tope de la cabeza",
+    "wearables_skin": "Skin",
```
Comment (LautaroPetaccio) [inline webapp/src/modules/translation/locales/es.json:91]: It seems that you're missing the category translation of skin.

File: webapp/src/components/HomePage/HomePage.tsx [hunk 2]
```diff
@@ -47,7 +47,7 @@ const HomePage = (props: Props) => {
 
   const sort: Partial<Record<View, SortBy>> = useMemo(
     () => ({
-      [View.HOME_NEW_ITEMS]: SortBy.RECENTLY_REVIEWED,
+      [View.HOME_NEW_ITEMS]: SortBy.RECENTLY_LISTED,
```
Comment (LautaroPetaccio) [inline webapp/src/components/HomePage/HomePage.tsx:50]: Would you mind explaining the reason behind this change?

File: webapp/.ci/.env.stg [hunk 3]
```diff
@@ -8,6 +8,7 @@ REACT_APP_ATLAS_SERVER_URL=https://api.decentraland.net
 REACT_APP_PEER_URL=https://peer-lb.decentraland.org
 REACT_APP_TRANSACTIONS_API_URL=https://transactions-api.decentraland.net/v1
 REACT_APP_BUILDER_URL=https://builder.decentraland.net
+REACT_APP_FF_SKINS=
```
Comment (LautaroPetaccio) [inline webapp/.ci/.env.stg:11]: Would you mind adding the FF in Vercel as well?

**Review by LautaroPetaccio — CHANGES_REQUESTED** [review]

File: webapp/src/components/HomePage/HomePage.tsx (hunk 2 above)
Comment (cazala) [reply webapp/src/components/HomePage/HomePage.tsx:50]: That's not related to this PR, it something that is currently broken in prod, if you click on the View More of the first slideshow, it takes you to the Collectibles page, with `sortBy=recently_reviewed` which is currently not a valid option for that page, so what happens is that the saga defaults to "recently_listed", but the dropdown in that page shows "Cheapest" as the selected option for some reason. With this change, the parameter is set to `sortBy=recently_listed` and the dropdown shows the right value "Recently Listed"

File: webapp/.ci/.env.stg (hunk 3 above)
Comment (cazala) [reply webapp/.ci/.env.stg:11]: Good catch! I added it ✅

**Review by LautaroPetaccio — APPROVED** [review]

**Pull request merged**